### PR TITLE
[Bug][Hans] redirect to home

### PR DESCRIPTION
**Pull request by HansGabriel**

### CODE REVIEW

File: hooks/useNavigate.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { useNavigation } from '@react-navigation/native';
+import { Routes } from '../enums/routes';
+
+export default function useNavigate(routeName: Routes) {
+  const navigation =
+    useNavigation<NativeStackNavigationProp<RootStackParamsList>>();
+
+  // TODO: use proper typing instead of never
+  return () => navigation.navigate(routeName as never);
+}
```
Comment (ezerssss) [inline hooks/useNavigate.ts:12]: seems like unnecessary abstraction? wats this for?

we can just import these two lines to what ever component and then call the `navigate` function

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: hooks/useNavigate.ts (hunk 1 above)
Comment (HansGabriel) [reply hooks/useNavigate.ts:12]: Um technically for navigation, we have to import three types and modules to declare navigation. 
For this, you just have to import one hook and call it

```ts
const navigateToHome = useNavigate(Routes.HOME);
```

Comment (ezerssss) [reply hooks/useNavigate.ts:12]: true, but dont u see this as an abstraction of an abstraction?

also this navigation na function only works for the `RootStack`, if we implement tabs it won't work na, pero wdyt?

Comment (HansGabriel) [reply hooks/useNavigate.ts:12]: sge we probably don't need an abstraction haha.

**Pull request closed without merging**